tsconfig.json

```diff
@@ -12,6 +12,7 @@
 
 		/* Strict Type-Checking Options */
 		"strict": true,                        /* Enable all strict type-checking options. */
+		"strictNullChecks": true,
```

Comment:
This should be enabled via `strict`, but I some `Type | undefined` types didn't seem to be handled correctly. Adding this seems to fix the issue.

I wonder if `strict` + `noImplicitAny` results in odd behavior.